### ISSUE TITLE
[false-positive] list(, $var) => "Use [...] instead"

### ISSUE THREAD

**Issue by ruudk** [kalessil/phpinspectionsea#107]
It complains that I should refactor this code:
```php
list(, $var) = doSomething();
```

to:
```php
[, $var] = doSomething();
```

But that's not valid. 

PS: Love this extension, you rock! :D

**Comment by kalessil** [comment]
Thank you @ruudk )

What is exactly invalid? Is it causing a PHP Runtime error or IDE complains to the new code fragment?

**Comment by ruudk** [comment]
Ok, I think my PHPStorm was not yet set to PHP 7.1 since it should just work:

https://3v4l.org/WjDpQ

**Comment by ruudk** [comment]
It's a bug in PHPStorm:

<img width="186" alt="screen shot 2017-01-13 at 16 06 46" src="https://cloud.githubusercontent.com/assets/104180/21934604/4fc05efc-d9aa-11e6-9626-c974477dba10.png">

**Comment by kalessil** [comment]
Thanks for checking ;)
